lib/Doctrine/DBAL/Platforms/AbstractPlatform.php

```diff
@@ -595,6 +595,16 @@ public function getBinaryDefaultLength()
     }
 
     /**
+     * Returns the maximum number of rows that can be inserted in a single INSERT statement.
+     *
+     * @return integer
+     */
+    public function getInsertMaxRows()
+    {
+        return 0;
```

Comment:
@edefimov not possible until a major version, as adding a new abstract method is a BC break.